presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/CachingHiveMetastore.java

```diff
@@ -1035,6 +1064,7 @@ public String toString()
             cacheBuilder = cacheBuilder.refreshAfterWrite(refreshMillis.getAsLong(), MILLISECONDS);
         }
         cacheBuilder = cacheBuilder.maximumSize(maximumSize);
```

Comment:
These can be chained together?
return cacheBuilder.maximumSize(maximumSize).recordStats();